docs/integration_fetching_data.md

```diff
@@ -63,6 +64,10 @@ async def async_setup_entry(hass, entry, async_add_entities):
             # handled by the data update coordinator.
             async with async_timeout.timeout(10):
                 return await api.fetch_data()
+        except ApiAuthError as err:
+            # Raising ConfigEntryAuthFailed with cancel future updates
```

Comment:
will cancel future updates

Reply:
fixed in https://github.com/home-assistant/developers.home-assistant/pull/881